.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,46 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
+
+- [ ] The PR follows our guidelines: 
+
+    - [Collaborator-Guide](https://github.com/expressjs/express/blob/master/Collaborator-Guide.md)
+    - [Contributing](https://github.com/expressjs/express/blob/master/Contributing.md)  
+    - [Release-Process](https://github.com/expressjs/express/blob/master/Release-Process.md)
+    
+- [ ] Tests for the changes have been added (for bug fixes / features)
+- [ ] Docs have been added / updated (for bug fixes / features)
+
+
+## PR Type
+What kind of change does this PR introduce?
+
+<!-- Please check the one that applies to this PR using "x". -->
+```
+[ ] Bugfix
+[ ] Feature
+[ ] Code style update (formatting, local variables)
```

Comment:
Adding on to @jonchurch's comment, we aren't generally accepting pull requests that just change coding style. Maybe it would be better to not list it here 🤔